src/main/types.d.ts

```diff
@@ -1,2 +0,0 @@
-declare module 'randomart';
```

Comment:
This was a weird one, originally I had to add these to get TS compiler to stop complaining that types couldn't be found for these modules, but then it started complaining again for some reason despite the existence of this file when I removed the transpile only false settings. 🤷 

It works and we're looking to rm these modules anyway so I'm rolling with the switch to require for now